### ISSUE TITLE
Add support for regions/ARIA landmark in VF framework

### ISSUE THREAD

**Issue by sandykadam** [visual-framework/vf-core#1830]
Currently, our VF is lacking regions that are helpful for accessibility to define the content region and also help screen readers.

We need to add regions like left, right, footer, sidebar, etc based on different content layouts or the ability to add roles.

Please check:
https://webaim.org/techniques/aria/#landmarks

https://www.w3.org/WAI/tutorials/page-structure/regions/

cc: @nikiforosk
